scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,9 @@
+# set nvcc as the underlying C++ compiler for Open MPI
+export OMPI_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set nvcc as the underlying C++ compiler for MPICH
+export MPICH_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
```

Comment:
Again - it says to use config here:

https://trilinos.org/about/documentation/building-with-cuda-support/